api/v1alpha1/oadp_types.go

```diff
@@ -395,6 +395,9 @@ func (dpa *DataProtectionApplication) AutoCorrect() {
 		if pvOperationTimeout, err := time.ParseDuration(resticTimeout); err == nil && dpa.Spec.Configuration.Velero.Args.PodVolumeOperationTimeout == nil {
 			dpa.Spec.Configuration.Velero.Args.PodVolumeOperationTimeout = &pvOperationTimeout
 		}
+		if dpa.Spec.Configuration.Velero.Args.RestoreResourcePriorities == "" {
```

Comment:
This might not trigger if user did not set any args to begin with.

See line 388
https://github.com/openshift/oadp-operator/pull/1132/files#diff-e186f96e971d9f53a2614ad247e98c7313a2fcc086cb9f6243d776e1e540fe73R388

Reply:
I would move it to after 401 before we remove duplicate values.

Reply:
Here you asked to remove `dpa.Spec.Configuration.Velero.Args != nil` from validation right? but current code does the same thing, right? Should we change or move it back to inside this if block to avoid checking same thing twice?